lesson4/tickets.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const app = express();
+app.listen(3618);
+app.use(express.static('public'));
+
+let count = 100;
+
+app.get('/santos',(req,res)=>{
+  const data = req.query.number;
+  if(data > count){
+ res.send('Not enough tickets')
+  }else{
+    count = count - data;
+    res.send('You bought: ' + (data + '') + 'tickets');
```

Comment:
you don't need data + '', and you don't need brackets. Try to rethink the mechanism why number + '' converts it. Don't just memorize the code and 'paste' it